inspector/src/components/actionTabs/tabs/performanceViewer/performanceViewerComponent.tsx

```diff
@@ -76,10 +70,20 @@ export const PerformanceViewerComponent: React.FC<IPerformanceViewerComponentPro
         if (window) {
             window.close();
         }
+        setIsLoadedFromCsv(false);
         setIsOpen(false);
-        setIsLoaded(false);
     };
 
+    useEffect(() => {
+        if (!isLoadedFromCsv) {

```

Comment:
what is the flow when setting this to true?
This seems to be a cleanup when setting it to false (the default behavior), but shouldn't be a cleanup when setting it to true?

Reply:
I wanted this useEffect to run only on the case where isLoadedToCsv's value changed from true to false (so, closing a window that was opened from a CSV file). When this happens, the performance collector should clear its data so it's ready to open either another CSV or a live recording.